Signal/src/ViewControllers/NewGroupViewController.m

```diff
@@ -206,7 +206,7 @@ - (void)updateTableContents
             [nonContactsSection
                 addItem:[OWSTableItem itemWithCustomCellBlock:^{
                     NewGroupViewController *strongSelf = weakSelf;
-                    OWSAssert(strongSelf);
+                    OWSCAssert(strongSelf);
```

Comment:
We need to use the "C" flavor of the Assert and Fail macros inside blocks since the non-C flavors capture self.